GPTResponder.py

```diff
@@ -42,12 +47,17 @@ def generate_response_from_transcript_no_check(self, transcript):
 
         except Exception as exception:
             print(exception)
-            return ''
+            root_logger.error('Error when attempting to get a response from LLM.')
```

Comment:
I like that we are starting to add exception messages to logs. Helpful with debugging errors